### ISSUE TITLE
URL broken

### ISSUE THREAD

**Issue by daisylab** [IBM/LNN#74]
Hi.

I'm a big fan of LNN, and this github site is a great pleasure to me.

However, recently, the link to the documentation is broken, which is the section "Documentation" of the `README.md`.

The link is 

```
https://ibm.github.io/LNN/introduction.html
```

but it says http 404 error.

**Comment by DuncanCowell** [comment]
Came in to say the same, thanks to @daisylab for opening the issue.

**Comment by arifammar93** [comment]
Agreed. Well organized documentation would help a great deal in furthering research into LNNs. 

Im working on a thesis that will be using LNNs and missing tutorials and lack of documentation is a huge impedance. 

Great work nonetheless! Thank you :)

**Comment by NaweedAghmad** [comment]
Thanks for bringing this to our attention everyone. We just pushed a big update that included a restructuring of the docs, we'll be patching this up shortly.
